regulation/node.py

```diff
@@ -73,7 +73,7 @@ def to_json(self):
         node_dict['text'] = self.text
         if self.title and self.title != '':
             node_dict['title'] = self.title
-        if self.marker:
+        if self.marker is not None:
```

Comment:
What happens on else? Is marker pre-set to "" somewhere not shown in this snippet?